src/lib.rs

```diff
@@ -227,11 +227,11 @@ pub trait Effectful {
         Boxed::new(self)
     }
 
-    /// Run this computation to completion on the current thread
+    /// Run the computation to completion on the current thread
     ///
-    /// This method blocks the current thread while waiting for the progress of the computation
+    /// This method blocks the current thread while waiting on the progress of the computation
     ///
-    /// The effect type of this computation must be an empty set (never type) since there is no handler
+    /// The effect type of this computation must be an empty set (never typed) since there is no handler
```

Comment:
Here, *never type* means the special uninhabited type `!` which denotes an empty set in the library.